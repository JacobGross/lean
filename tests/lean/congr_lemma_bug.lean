constant P : Type₁
constant P_sub : subsingleton P
attribute P_sub [instance]
constant q : P → Prop

#congr q
