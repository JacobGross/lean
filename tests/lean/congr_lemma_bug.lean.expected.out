[cast]
λ (a a_1 : P), eq.trans (eq.refl (q a)) (congr (eq.refl q) (subsingleton.elim a a_1))
:
∀ (a a_1 : P), q a = q a_1
